resources/views/layouts/navbar.blade.php

```diff
@@ -20,20 +20,31 @@
         </div>
 
         <b-navbar-nav class="d-flex align-items-center">
-            @foreach(Menu::get('topnav')->items as $item)
-                <b-nav-item href="{{ $item->url() }}" {{$item->isActive !== false ? 'active': ''}}>
-                    {!! $item->title !!}
-                </b-nav-item>
-            @endforeach
-
             @if(Menu::get('customtopnav'))
-            @foreach(Menu::get('customtopnav')->items as $item)
-                <li class="nav-item">
```

Comment:
Can we change this back to use the b-nav-item? https://bootstrap-vue.org/docs/components/navbar#navbar

Reply:
Fixed, changed to <b-nav-item>